src/corehost/cli/fxr/hostfxr.cpp

```diff
@@ -60,14 +60,18 @@ int execute_app(
 
     if (code != StatusCode::Success)
     {
-        trace::error(_X("Could not load host policy library from [%s]"), impl_dll_dir.c_str());
+        trace::error(_X("Expected to load %s from [%s]"), LIBHOSTPOLICY_NAME, impl_dll_dir.c_str());
```

Comment:
I would suggest a more user-understandable message - "Invalid .NET Core FX configuration in folder <path to folder>"